inst/tinytest/test-guide_prism_bracket.R

```diff
@@ -53,7 +57,7 @@ expect_silent(ggplotGrob(g2))
 control <- grab_axis(g1, side = "l")
 test <- grab_axis(g2, side = "l")
 
-p + scale_x_discrete(position = "right", guide = "prism_bracket")
+# p + scale_x_discrete(position = "right", guide = "prism_bracket")
```

Comment:
I wasn't sure what this part was supposed to do. It's unfamiliar to me to place an x-axis on the right, and this gives an error with development version of ggplot2.